server/chat-plugins/mafia-data.js

```diff
@@ -390,7 +402,7 @@ const roles = {
 	coroner_cracking_idea: `coronercrackingidea`,
 	coronercrackingidea: {
 		name: `Coroner (Cracking Idea)`,
-		id: `coronercrackingidea`,
+		id: `coronerci`,
```

Comment:
`id` should be the ID version (all lowercase, no spaces or symbols, keep numbers) of the property (eg: `coronercrackingidea`).